docs/.gitignore

```diff
@@ -1,3 +1,5 @@
+**/.DS_Store
+Gemfile.lock
```

Comment:
Nope, this must be checked in to do its job (locking down exact dependency versions so that builds are reproducible)

Reply:
Thanks. fixed in both PRs.